src/CONST.js

```diff
@@ -132,6 +132,7 @@ const CONST = {
     STAGING_SECURE_URL: 'https://staging-secure.expensify.com/',
     NEWDOT: 'new.expensify.com',
     NEW_EXPENSIFY_URL,
+    NEW_EXPENSIFY_URL_STAGING: 'https://staging.new.expensify.com',
```

Comment:
NAB really small nitpick but we have `STAGING_SECURE_URL` and most of these URL constants ending in `URL` maybe we can make this `STAGING_NEW_EXPENSIFY_URL` for consistency?